### ISSUE TITLE
Is it possiable to start get post data with certain time

### ISSUE THREAD

**Issue by Aaron-X-93** [kevinzg/facebook-scraper#476]
I am trying to collect data under an account with thousands of posts. The program got some error from time to time so each time I have to start from the beginning.

I am wondering if the  library is able to set certain time or date for starting get the posts. For example, get all posts before Jan. 01, 2021 of an account. And if possible, how to do that? Thanks.

**Comment by neon-ninja** [comment]
Yes, you can use the `start_url` parameter. See https://github.com/kevinzg/facebook-scraper/issues/310#issuecomment-852652846

**Comment by Aaron-X-93** [comment]
Thanks, it works.

**Comment by tridelt** [comment]
is it also possible to start from a point in time onward without having the url?
I only need posts between 2021 and 2022.
Thanks.
